.github/workflows/publish_timestamped_release.yml

```diff
@@ -5,6 +5,7 @@ on:
     branches:
       - master
       - stage-swan-lake
+      - ballerina-2.0.x
```

Comment:
We'll have to remove the workflow configs of ballerina-2.0.x

Reply:
Removed in b45bcf1.